server/src/main/java/org/room/apollo/server/entity/User.java

```diff
@@ -0,0 +1,88 @@
+package org.room.apollo.server.entity;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+/**
+ * Created by Alexey on 10/8/17.
+ */
+@Document(collection = "users")
+public class User {
+    @Id
+    private String id;
+    @Indexed(unique = true)
+    private String username;
+    private String password;
```

Comment:
Are we going to store password in User's entity? Maybe there is an authentication framework out-of-the-box?
Cause in this case, if I understand right, developer will be able to retrieve user's password through database access.